src/common/libkvs/kvs_txn.h

```diff
@@ -7,6 +7,10 @@ extern "C" {
 
 #include <stdarg.h>
 
+enum {
```

Comment:
Just noticed all the flags in kvs.h are now lookup specific.  I moved them into kvs.h because earlier we alloed the FLUX_KVS_TREEOBJ flag with txn_put.

I guess I'd vote we put all the KVS user flags in the same enum in kvs.h unless others feel they should be split up.

Reply:
ahh, sounds good to put it all in kvs.h